app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -1012,4 +1017,8 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Session
         private const val REQUEST_CODE_PROMPT_PERMISSIONS = 2
         private const val REQUEST_CODE_APP_PERMISSIONS = 3
     }
+
+    override fun onAccessibilityStateChanged(enabled: Boolean) {
+        browserToolbarView.setScrollFlags(enabled)
```

Comment:
could you verify on-device that this works for bottom toolbar? Looking at `setScrollFlags` it look like it might early return if `shouldUseBottomToolbar` before disabling scroll? Didn't test this myself yet though 🤔

Reply:
Yes, but if the toolbar is at bottom the flags weren't set in the first place, so there is no need to update? 🤔

Reply:
So is it not possible to disable the toolbar hiding on the bottom toolbar? Sorry I don't know much about these toolbar behaviors 😓

Reply:
AFAIK bottom toolbar uses a custom `BrowserToolbarBottomBehavior` to scroll away, while top toolbar uses these flags  to scroll, and they are updated and the toolbar is expanded when accessibility services are enabled. So these flags are set and updated only for top toolbar.

Reply:
So ideally, the bottom toolbar would also be able to be static for talkback users/dynamic otherwise in the same way as the top toolbar, but we can file a followup for this (there might be a very good reason why we can't do that right now)